troposphere/static/js/components/images/detail/availability/AvailabilityView.jsx

```diff
@@ -36,9 +29,12 @@ export default React.createClass({
         let optMachineID;
         if (!isSummary) {
             optMachineID = (
-                <Code mb="10px">
-                    <div onClick={ this.onClick }>{ machineID }</div>
-                </Code>
+                <div>
+                    <Code mb="10px">
+                        { machineID }
+                    </Code>
+                    <CopyButton text={ machineID }/>
```

Comment:
👍 I like this idea of being able to explicitly state what text will be copied.

This would allow us to do things like "Pretty-print" inside the div and have the "raw copy text" available for CopyButton. 😄